### PR TITLE
Add ExecutorServiceFullCore snippet

### PR DESCRIPTION
**Pull request by grzesiekkedzior**

### CODE REVIEW

File: src/test/java/thread/ExecutorServiceFullCoreTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+class ExecutorServiceFullCoreTest {
+  /**
+   * Tests for {@link ExecutorServiceFullCore#getExecutorFullCore()}.
+   */
+  @Test
+  public void testExecutorServiceFullCore() {
+    var core = Runtime.getRuntime().availableProcessors();
+    ThreadPoolExecutor executorService =
+            (ThreadPoolExecutor) ExecutorServiceFullCore.getExecutorFullCore();
+    assertEquals(core, executorService.getCorePoolSize());
```
Comment (iluwatar) [inline src/test/java/thread/ExecutorServiceFullCoreTest.java:47]: Check the parameters. I think its `expected, actual`, not the other way around.

File: src/test/java/thread/ExecutorServiceFullCoreTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    assertNotEquals(core + 1, executorService.getCorePoolSize());
```
Comment (iluwatar) [inline src/test/java/thread/ExecutorServiceFullCoreTest.java:48]: Maybe check against 0 as well.

**Review by iluwatar — CHANGES_REQUESTED** [review]

File: src/test/java/thread/ExecutorServiceFullCoreTest.java (hunk 1 above)
Comment (grzesiekkedzior) [reply src/test/java/thread/ExecutorServiceFullCoreTest.java:47]: I know but number of processors return jvm. The test for any other machines will not pass when I use a concrete number. Expected is jvm value from availableProcessors method. Actual is the size of thread pool which is created by this value.

Comment (iluwatar) [reply src/test/java/thread/ExecutorServiceFullCoreTest.java]: Yes, you're right

File: src/test/java/thread/ThreadPoolTest.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+ * Tests for 30 Seconds of Java code library
+ *
+ */
+class ThreadPoolTest {
+  /**
+   * Tests for {@link ThreadPool#createFixedThreadPool()}.
+   */
+  @Test
+  public void testCreateFixedThreadPool() {
+    int numProcessors = Runtime.getRuntime().availableProcessors();
```
Comment (iluwatar) [inline src/test/java/thread/ThreadPoolTest.java:44]:
```suggestion
    var numProcessors = Runtime.getRuntime().availableProcessors();
```

File: src/test/java/thread/ThreadPoolTest.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    ThreadPoolExecutor executorService =
```
Comment (iluwatar) [inline src/test/java/thread/ThreadPoolTest.java:45]:
```suggestion
    var executorService =
```

File: src/test/java/thread/ThreadPoolTest.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            (ThreadPoolExecutor) ThreadPool.createFixedThreadPool();
+    assertEquals(numProcessors, executorService.getCorePoolSize());
+    assertNotEquals(0, ThreadPool.createFixedThreadPool());
```
Comment (iluwatar) [inline src/test/java/thread/ThreadPoolTest.java:48]:
```suggestion
    assertNotEquals(0, executorService.getCorePoolSize());
```

**Review by iluwatar — CHANGES_REQUESTED** [review]

**Review by iluwatar — APPROVED** [review]

**Pull request merged**